src/app/hooks/useStxTransactionRequest.ts

```diff
@@ -0,0 +1,41 @@
+import { txPayloadToRequest } from '@secretkeylabs/xverse-core';
+import { deserializeTransaction } from '@stacks/transactions';
+import { decodeToken } from 'jsontokens';
+import { useLocation } from 'react-router-dom';
+
+const useStxTransactionRequest = () => {
```

Comment:
would be good to have a return type here. related to the core code review on txPayloadToRequest

Reply:
There's an issue with strictly typing this hook since the types that are exposed from the `@stacks/connect` lib have errors with the types themselves so until that's resolved loosely typing here is our only solution.

I will open an issue on the `@stacks/connect` repo and add a follow-up issue.